src/io/file.c

```diff
@@ -20,7 +20,8 @@ int cr_file_match_str(FILE *f, const char *str)
     }
 
     /* consume the rest of what's available */
-    while (fread(buf, 1, sizeof (buf), f) > 0) ;
+    if (read)
+      while (fread(buf, 1, sizeof (buf), f) > 0) ;
```

Comment:
small nit: please indent that by 4 spaces.

Reply:
Should be fixed now :+1: Thanks!